.codegen/gateway/gloo-edge/settings.yaml

```diff
@@ -20,13 +20,13 @@ spec:
       replaceInvalidRoutes: false
     disableKubernetesDestinations: false
     disableProxyGarbageCollection: false
-  discoveryNamespace: nukleros-gateway-system
+  discoveryNamespace: nukleros-gateway-system # +operator-builder:field:name=namespace,default="nukleros-gateway-system",type=string
   kubernetesArtifactSource: {}
   kubernetesConfigSource: {}
   kubernetesSecretSource: {}
   refreshRate: 60s
-
   gateway:
+    persistProxySpec: true
```

Comment:
This config setting is required to have Gloo persist its internal `Proxy` objects to the etcd custom resource. This is what's consumed by external dns to map hostnames to lb endpoints.